docker/dev/deploy.sh

```diff
@@ -6,10 +6,11 @@ new_tag="$1"
 git tag $new_tag
 
 # Setting the version on the front end
-pushd ../../ui/
-node set_version.js
-git add src/version.ts
-popd
+echo "export let version: string = '$(git describe --tags)';" > "ui/src/version.ts"
+git add "ui/src/version.ts"
+# Setting the version on the backend
+echo "pub const VERSION: &'static str = \"$(git describe --tags)\";" > "server/src/version.rs"
+git add "server/src/version.rs"
```

Comment:
Nice! This is a great way of abstracting that. Not sure why I didn't think to do this in bash instead of a pointless node script.